src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -276,10 +277,77 @@ void IslandWindow::OnSize(const UINT width, const UINT height)
     }
 }
 
+// Method Description:
+// - Handles a WM_GETMINMAXINFO message, issued before the window sizing starts.
+//   This message allows to modify the minimal and maximal dimensions of the window.
+//   We focus on minimal dimensions here
+//   (the maximal dimension will be calculate upon maximizing)
+//   Our goal is to protect against to downsizing to less than minimal allowed dimensions,
+//   that might occur in the scenarios, where _OnSizing is bypassed.
+//   An example of such scenario, is anchoring the window to the top/bottom screen border

```

Comment:
```suggestion
//   that might occur in the scenarios where _OnSizing is bypassed.
//   An example of such scenario is anchoring the window to the top/bottom screen border
```

extraneous commas

Reply:
I will never learn this - in my mother tongue the comma is required.. while in English it is not.